src/server/services/item_manager.js

```diff
@@ -1,102 +1,88 @@
 const fs = require("fs").promises;
 const pokemon_client = require("../clients/pokemon_client");
+const { Items } = require("../db/models");
 
 const pokemonClient = new pokemon_client();
-const todoFile = "todo_list.json";
 
 async function addTodoItem(input) {
   const inputString = input.trim();
-  let dataArray = await getTodoItems();
 
-  if (!dataArray) dataArray = [];
   // number
   if (/^[0-9]+$/.test(inputString)) {
-    handlePokemonItem(inputString, dataArray);
+    handlePokemonItem(inputString);
```

Comment:
consider changing to a more descriptive name + more different from addPokemonItem